### PR TITLE
Fixed few things

### PR DESCRIPTION
**Pull request by Aisu2635**
## Title :
slight basic changes to images

## Description
so i noticed on the developed projects section on mobile view the images were not rounded so i fixed it
adjusted the aspect ratios of all the images in this section so all have same width & height 
also slight better quality pics uploaded.
updated the readme.md so that anyone can create a .env file for local hosting 
> dont worry I havent leaked API key hehehe.

## Screenshots (if applicable)
image

## Checklist
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/tcet-opensource/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] I have read and followed the **Contributing Guidelines**

### CODE REVIEW

**Comment by kamalika0363** [comment]
image

### In this slider can you try to increase the scale of image and make the side arrows a little small.

File: blog/2023-10-1-cloudgaming/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+slug: Cloud-gaming
+title: Exploring the Future of Gaming
+author: [Om Hinge]
```
Comment (vish-rt) [inline blog/2023-10-1-cloudgaming/index.md:4]: This line is breaking the deployment.

That's not how you add authors. Check the posting guidelines blog for details.

**Review by vish-rt — CHANGES_REQUESTED** [review]
Check the comments and fix it. I recommend raising a separate PR for the blog which goes to the `blogs` branch.

**Comment by Aisu2635** [comment]
tried to change the image size but couldnt figure out the buttons

File: blog/2023-10-1-cloudgaming/index.md (hunk 1 above)
Comment (Aisu2635) [reply blog/2023-10-1-cloudgaming/index.md]: ohhk

**Review by vish-rt — APPROVED** [review]
My changes resolved. @kamalika0363 you can merge once your changes are approved

**Pull request merged**